.github/workflows/check+deploy.yaml

```diff
@@ -0,0 +1,61 @@
+on:
+    push:
+        branches: [ master ]
+
+    pull_request:
+
+name: Check & Deploy
+
+jobs:
+    check-and-deploy:
+        name: Check & Deploy on Ubuntu (latest)
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout git repository
+              uses: actions/checkout@v2
+              with:
+                submodules: true
+                path: _src/
+
+            - name: Install dependencies
+              shell: bash
+              run: |
+                pushd _src
+                python3 -m pip install --upgrade 'numpy>=1.16.5' 'scipy>=1.0' pandas 'pylha>=0.2' pyyaml 'ckmutil>=0.3.2' 'rundec>=0.5' voluptuous nose
```

Comment:
Can't this be skipped? If a dep is missing in setup.py, we wanna catch it

Reply:
You are absolutely right! This was part of my first attempt, when I did not yet install ``wilson`` using pip, but rather tested in-place with ``nose``. Going to remove it!

Reply:
Except for nose, which we need for the tests.